COMMITTERS.md

```diff
@@ -34,6 +35,10 @@ This file details what is needed for different operations in the repository that
 
 Before providing a fixed package version, you need to determine the existing version specified in the repository, what type of version it uses, and whether the package contains additional handling for creating fixed versions.
 
+### Updating a Manual Package
+
+A manual package is one that is not using the Chocolatey-AU automation. The specific package files should be updated normally (i.e. the `.nuspec` file and the Chocolatey package scripts). Once those are updated, and the pull request merged, the package is pushed to the Chocolatey Community Repository by makeing an empty commit, with the mesage int he format `[PUSH <PACKAGENAME>]` where `<PACKAGENAME>` is the name of the package you want to push.
```

Comment:
```suggestion
A manual package is one that is not using the Chocolatey-AU automation. The specific package files should be updated normally (i.e. the `.nuspec` file and the Chocolatey package scripts). Once those are updated, and the pull request merged, the package is pushed to the Chocolatey Community Repository by makeing an empty commit, with the mesage in the format `[PUSH <PACKAGENAME>]` where `<PACKAGENAME>` is the name of the package you want to push.
```

Reply:
In addition to what @corbob has suggested, can we change `mesage` to be `message`?

Reply:
I've added @gep13's suggestion into the original suggestion.

Reply:
Apologies for my abysmal spelling. Fixed.